pkg/keccak/Makefile.dep

```diff
@@ -0,0 +1,5 @@
+FEATURES_BLACKLIST += arch_8bit arch_16bit arch_mips32r2 arch_riscv
```

Comment:
Sorry for the round trips. In fact there's a mechanism for explicitly enable for any of one element a list of features: `FEATURES_REQUIRED_ANY`.
Although it's an ARM, the lpc2387 is not supported but the keccak build system apparently, so it has to be blacklisted.

```suggestion
FEATURES_REQUIRED_ANY += arch_arm|arch_native
FEATURES_BLACKLIST += arch_arm7
```

Reply:
Wouldn't `FEATURES_BLACKLIST += arch_arm7` blacklist much more than that board?

Reply:
Yes but that wouldn't work anyway. excluding the arch doesn't require to list all boards using a CPU of that arch.

Reply:
Apologies, but I'm not sure if there is a misunderstanding here somewhere. You suggested I add `FEATURES_BLACKLIST += arch_arm7`, which I thought would be too much. What exactly is it that you say wouldn't work?

Reply:
`arch_arm7` is only used by the lpc2387 cpu and only a few boards use this CPU: msba2, mcb2388 and avsextrem. The support is different from the cortexm but it's still an ARM (arch_arm).
So it's covered by `FEATURES_REQUIRED_ANY += arch_arm|arch_native` but it doesn't work with the toolchain keccak. So `arch_arm7` has to be blacklisted explicitly.

Reply:
I see, thank you for the clarification.